### PR TITLE
Update

### PR DESCRIPTION
**Pull request by Schweikhard**
I've added the original files into raw and edited the metadata and contributors files.
What's mainly missing now is the lexibank.py

### CODE REVIEW

**Review by LinguList — COMMENTED** [review]
I guess, it should be enough to have the list of languages, and the lingpy-tsv file in the repository, as we'll re-create the cldf anyway. We also need the concepticon file. These need to be placed in a folder `etc/concepts.tsv` and `etc/languages.tsv`, but I think even that the concept list is identical with `Blust-2008-210` in which case it is enough to actually place this information into `metadata.json`. The source is also important, and the sources for individual language in `polynesian.bib` can also sttay there, but the big json file is not needed. As contributor, `Mary Walworth` should be added, her role being `data curation, cognate coding`.

**Comment by Schweikhard** [comment]
The IDs differ, but otherwise it's the same as Blust-2008-210
E.g. Blust-2008-210-175 has ID 1741 in parameters.csv

File: metadata.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
-    "title": "walworthpolynesian",
-    "url": "",
-    "license": "",
-    "conceptlist": "",
-    "citation": ""
+    "title": "Sequence Comparison in Computational Historical Linguistics",
```
Comment (LinguList) [inline metadata.json:2]: As title, I recommend: `Refined Wordlists of East Polynesian Languages`

File: raw/orthography.tsv [hunk 2]
```diff
@@ -0,0 +1,127 @@
+Grapheme	IPA	NOTE
```
Comment (LinguList) [inline raw/orthography.tsv:1]: Move this to `etc/orthography.tsv`

File: metadata.json [hunk 3]
```diff
@@ -1,7 +1,7 @@
...
+    "url": "https://zenodo.org/record/1252231",
+    "license": "http://www.opensource.org/licenses/GPL-3.0",
+    "conceptlist": "Blust-2008-210",
+    "citation": "Johann-Mattis List, Mary Walworth, Simon J. Greenhill, Tiago Tresoldi, & Robert Forkel. (2018, May 24). LingPy Tutorial. Version 1.0 (Version v1.0). Journal of Language Evolution. Zenodo. http://doi.org/10.5281/zenodo.1252231"
```
Comment (LinguList) [inline metadata.json:6]: Citation should quote the original paper, as it is common, not the tutorial.

**Review by LinguList — CHANGES_REQUESTED** [review]

**Pull request merged**